jobs/opensearch/templates/config/opensearch-security/roles_mapping.yml.erb

```diff
@@ -22,10 +22,20 @@ own_index:
   - "*"
   description: "Allow full access to an index named like the username"
 
+index_management_full_access:
+  reserved: false
+  hidden: false
+  users:
+  - "logstash.opensearch.internal"
+  backend_roles:
+  - "logstash.opensearch.internal"
```

Comment:
I'm pretty sure this line isn't necessary, because the certificate for `logstash.opensearch.internal` should identify it as a user

Reply:
You were right